pkg/cmd/cli/download/describe.go

```diff
@@ -20,6 +21,7 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			logLevel := logLevelFlag.Parse()
 			logger := logging.DefaultLogger(logLevel, formatFlag.Parse())
+			logger.Debugf("setting log-level to %s", strings.ToUpper(logLevel.String()))
```

Comment:
Let's change "The command, datamgr download describe, is called " to "Datamgr download describe called"